services/openshift-service-impl/src/main/java/io/fabric8/launcher/service/openshift/impl/Fabric8OpenShiftServiceImpl.java

```diff
@@ -318,7 +319,10 @@ public boolean projectExists(String name) {
         if (name == null || name.isEmpty()) {
             throw new IllegalArgumentException("Project name cannot be empty");
         }
-        return controller.checkNamespace(name);
+        return client.projects().list().getItems().stream()
```

Comment:
This doesn't work with the starter clusters. If you look at the history, we used to do that same code.

Reply:
See https://github.com/fabric8-launcher/launcher-backend/issues/459 for more information